config/common.go

```diff
@@ -35,3 +43,13 @@ func GetMinimumQuestionsCountPerShard(status *guard.FileStoreStatus) (val int) {
 	//return int(status.FileStoreMeta.CheckFrequency) * ConstMinQuestionsCountPerChallenge
 	return ConstMinQuestionsCountPerShard
 }
+
+func InitDB() {
+	//get db and redis connection strings
+	DbStatusURL, FoundPgStatusString = os.LookupEnv("TEST_DB_URL_STATUS")
+	DbGuardURL, FoundPgGuardString = os.LookupEnv("TEST_DB_URL_STATUS")
+	RdURL, FoundDbRdString = os.LookupEnv("TEST_RD_URL")
+	if FoundPgStatusString == false || FoundPgGuardString == false || FoundDbRdString == false {
+		log.Error(fmt.Sprintf("TEST_DB_URL_STATUS or TEST_DB_URL_STATUS or TEST_RD_URL env vars need to be set before running test"))
```

Comment:
This does not return error back to the caller, so the test cannot actually fail with a log.Error

Reply:
okay. returned error or nil to the calling function.